stub/src/main/java/io/grpc/stub/RpcMethod.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.stub;
+
+import io.grpc.MethodDescriptor.MethodType;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotates a method descriptor method to provide metadata for annotation processing.
+ */
+@Retention(RetentionPolicy.CLASS)
```

Comment:
We would normally annotate this initially with `@ExperimentalApi`. That might cause some problem with a experimental api checker. @ericgribkoff can help figure out what to do here.

Reply:
Added annotations, we can see if it needs to be removed.

Reply:
Yes, as-is this will trigger the grpc-java-api-checker tool to warn that user code (the generated code) is invoking `@ExperimentalApi`.

@ejona86 If other libraries are shipping gRPC's generated code, how does annotating part of that code as "experimental" work? If we remove/rename this class in the future, the previously generated code will break.

Reply:
Yeah, I agree. It can't be experimental because the generated code won't compile without it. I realized that yesterday. The earlier issue we had with the MethodDescriptors was a special case.

Reply:
Removed the annotation.